content/ja/handbook/customer-success/csm/escalations/index.md

```
@@ -0,0 +1,350 @@
+stale: false
+---
+
+追加の CSM 関連ハンドブックページは [CSM ハンドブックホームページ](/handbook/customer-success/csm/)をご覧ください。
+
+---
+
+## 背景
+
+エスカレーションには少なくとも 2 つの異なる形態があります:
+
+1. アカウントエスカレーション。これは、顧客が課題を表明した場合、または GitLab チームメンバーが顧客が困難な状況に直面していることを特定した場合に発生します。これは特定のサポートチケットに関連している場合もそうでない場合もあります。GitLab の誰でも[アカウントエスカレーションを開始](#opening-the-escalation)し、[適切なグループのリーダーシップに回覧して DRI を見つける](#escalation-dri)ことができます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)